data/places.ts

```diff
@@ -668,5 +668,35 @@ export const PLACES: IPin[] = [
     coordinates: [43.26925725801514, -2.93479144050948],
     date: '2024-08-04',
     photo: '/photos/voidbert/bilbao.jpg'
+  },
+  {
+    author: 'João Lobo',
+    username: 'joaodiaslobo',
+    type: EPinType.Sticker,
+    city: 'Milan',
+    country: 'Italy',
+    coordinates: [45.465573884554935, 9.196202807173558],
+    date: '2024-08-23',
+    photo: '/photos/joaodiaslobo/milan.jpeg'
+  },
+  {
+    author: 'João Lobo',
+    username: ['joaodiaslobo', 'feliciofilipe'],
```

Comment:
Where's Felício on the photo? 😅

Reply:
This is a very fun story! 👀
While I was walking through Florence and simultaneously looking for spots to place a CeSIUM sticker, I found a very old and yellowish one that wasn't marked on our map! So I went ahead, took a photo, and contacted multiple people to solve this mystery.
I found out that this sticker was actually placed by Felício way back in 2021! But back then he lost the photos of his Italy trip and so he never got to place them on CeSIUM Pelo Mundo...
I have since talked to him about this and he gave me permission to steal this sticker and place it as one of my own! 👑
His username is there so that it gets added to his Pins count and to credit him. ✌️

Reply:
Damn, that was unexpected 😁